app/views/shared/my_facilities/_ranked_facilities.html.erb

```diff
@@ -0,0 +1,234 @@
+      <td class="type-percent"><em>45%</em></td>
+      <td class="type-number">18</td>
+      <td class="type-number">451</td>
+      <td class="type-number">1321</td>
+      <td class="type-number">12</td>
+      <td class="type-rank">3 <i class="fas fa-long-arrow-alt-up"></i></td>
+    </tr>
+    <tr>
+      <td class="type-rank">13</td>
+      <td class="type-title"><a href="#">CHC Sparrow Farm</a></td>
+      <td class="type-percent"><em>51%</em></td>
+      <td class="type-number">121</td>
+      <td class="type-number">1,908</td>
+      <td class="type-number">1,234</td>
+      <td class="type-number">9</td>
+      <td class="type-rank">3 <i class="fas fa-long-arrow-alt-up"></i></td>
+    </tr>
+    <tr>
+      <td class="type-rank">14</td>
+      <td class="type-title"><a href="#">CHC Elephant Lake</a></td>
+      <td class="type-percent"><em>42%</em></td>
+      <td class="type-number">125</td>
+      <td class="type-number">809</td>
+      <td class="type-number">1234*</td>
+      <td class="type-number">12</td>
+      <td class="type-rank">15 <i class="fas fa-long-arrow-alt-up"></i></td>
+    </tr>
+    <tr>
+      <td class="type-rank">15</td>
+      <td class="type-title"><a href="#">CHC Tiger and Dolphin Club</a></td>
+      <td class="type-percent"><em>51%</em></td>
+      <td class="type-number">102</td>
+      <td class="type-number">900</td>
+      <td class="type-number">1,234</td>
+      <td class="type-number">12</td>
+      <td class="type-rank">17 <i class="fas fa-long-arrow-alt-up"></i></td>
+    </tr>
+    <tr>
+      <td class="type-rank">16</td>
+      <td class="type-title"><a href="#">CHC Squirrel in a Tree</a></td>
+      <td class="type-percent"><em>51%</em></td>
+      <td class="type-number">123</td>
+      <td class="type-number">2,345</td>
+      <td class="type-number">4,567*</td>
+      <td class="type-number">12</td>
+      <td class="type-rank">23 <i class="fas fa-long-arrow-alt-up"></i></td>
+    </tr>
+    <tr>
+      <td class="type-rank">17</td>
+      <td class="type-title"><a href="#">CHC Hippopotamus Mudplex</a></td>
+      <td class="type-percent"><em>23%</em></td>
+      <td class="type-number">123</td>
+      <td class="type-number">1,456</td>
+      <td class="type-number">12,345</td>
+      <td class="type-number">456</td>
+      <td class="type-rank">24 <i class="fas fa-long-arrow-alt-up"></i></td>
+    </tr>
+    </tbody>
+  </table>
+</div>
+<section class="footnotes">
+    <ol>
+        <li><strong>Ranking:</strong> is ranked by...</li>
+        <li><strong>BP control:</strong> Had at least one visit in last 90 days and BP was &lt;140/90 at last visit.</li>
+        <li><strong>OPD load:</strong> Outpatient department load is entered by surveillance officers for each facility. Monthly OPD load is an average. Where a facility has no OPD Load entered (see asterisk), the number is an average of other faciities of the same type in this district.</li>
+    </ol>
+</section>
```

Comment:
A couple of sneaky missing newlines, hehe. I'll let you sort that out with CodeClimate. You could also run the following locally to auto-fix

```
$ gem install rubocop
$ rubocop --auto-correct --only 'Layout/TrailingEmptyLines'
```